packages/block-editor/src/components/block-list/style.scss

```diff
@@ -448,7 +448,7 @@
 	// Left
 	&[data-align="left"] {
 		// This is in the editor only; the image should be floated on the frontend.
-		.block-editor-block-list__block-edit {
+		> .block-editor-block-list__block-edit {
```

Comment:
Why do we need this change? I think the changes in the other files fix the issue for me.

Reply:
@youknowriad That's right! Something goes wrong with the pull request. I don't know why the files from an older pull request are included too. 

The changes in "packages/block-library/src/media-text/style.scss" fix this issue.

Reply:
Happy to merge the PR if you can manage to clean it :)

Reply:
Not sure how I can edit this pull request to remove the part "e0d6636"?